pkg/controller/virtualmachinevolumeexport/exporter_pod.go

```diff
@@ -0,0 +1,158 @@
+package virtualmachinevolumeexport
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+	hc "kubevirt-image-service/pkg/apis/hypercloud/v1alpha1"
+	vmv "kubevirt-image-service/pkg/controller/virtualmachinevolume"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	// SourceVolumeName provides a const to use for source volume in pod specs
+	SourceVolumeName = "source-volume"
+	// ExportVolumeName provides a const to use for export volume in pod specs
+	ExportVolumeName = "export-volume"
+	// SourceDevicePath provides device path for source volume in pod specs
+	SourceDevicePath = "/dev/source-block-volume"
+	// SourceDataDir provides mount path for source volume in pod specs
+	SourceDataDir = "/source"
+	// ExportDataDir provides mount path for export volume in pod specs
+	ExportDataDir = "/export"
+	// ExporterSourcePath  provides a constant to capture our env variable "EXPORTER_SOURCE_PATH"
+	ExporterSourcePath = "EXPORTER_SOURCE_PATH"
+	// ExporterExportDir  provides a constant to capture our env variable "EXPORTER_EXPORT_DIR"
+	ExporterExportDir = "EXPORTER_EXPORT_DIR"
+	// ExporterDestination provides a constant to capture our env variable "EXPORTER_DESTINATION"
+	ExporterDestination = "EXPORTER_DESTINATION"
+	// ExporterName indicates container name in exporter pod
+	ExporterName = "kubevirt-image-service-exporter"
+	// ExporterImage indicates exporter container image
+	ExporterImage = "quay.io/tmaxanc/kubevirt-image-service-exporter:latest"
+	// ExporterDestinationLocal indicates Destaination to export is local
+	ExporterDestinationLocal = "local"
+	// ExporterDestinationS3 indicates Destaination to export is s3
+	ExporterDestinationS3 = "s3"
+)
+
+func (r *ReconcileVirtualMachineVolumeExport) syncExporterPod() error {
+	completed, found, err := r.isPvcCompleted()
```

Comment:
What does `Complete` mean? We need an explanation(with comment) on this.

Reply:
`completed` indicates if pvc export is completed. Comments added.